pvlib/test/test_ivtools.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May  9 10:51:15 2019
+
+@author: cwhanse
+"""
+
+import numpy as np
+import pandas as pd
+import pytest
+from pvlib import pvsystem
+from pvlib import ivtools
+from conftest import requires_scipy, requires_pysam
```

Comment:
I could not import this file into a Jupyter notebook (in order to get at the test data) without changing this line to
```python
from pvlib.test.conftest import requires_scipy, requires_pysam
```